### ISSUE TITLE
Replace MakerDAO logo in Connect environment to newbrand logo & name

### ISSUE THREAD

**Issue by CallmeT-ty** [powerhouse-inc/switchboard#266]

**Comment by froid1911** [comment]
@CallmeT-ty where the image? :-D

**Comment by CallmeT-ty** [comment]
![Sky Logomark](https://github.com/user-attachments/assets/beb8b78d-3bde-44c8-9829-ea70c84f42b6)
![Sky Logo Light](https://github.com/user-attachments/assets/71108b65-db8b-4e75-9c82-402b11649156)
But let's wait with uploading it as there might be some links floating around. It should only be visible on

Staging 
https://connect-staging.sky.money/ 
Production
https://connect.sky.money/  

Those url's don't share the same Drives as https://apps.powerhouse.io/makerdao/connect do they?

**Comment by froid1911** [comment]
I think this needs to be changed in Switchboard and not in Connect

**Comment by CallmeT-ty** [comment]
Yes, correct, sorry, Changed on switchboard so it becomes visible in connect.

**Comment by CallmeT-ty** [comment]
We'll also need to make that production drive for Sky to be set as the default drive that is loaded on Connect.sky.money

**Comment by froid1911** [comment]
https://apps.powerhouse.io/makerdao/switchboard/d/sky
